src/Induction/WellFounded.agda

```diff
@@ -54,12 +59,17 @@ module Some {a lt} {A : Set a} {_<_ : Rel A lt} {ℓ} where
   wfRec : SubsetRecursor (Acc _<_) (WfRec _<_)
   wfRec = subsetBuild wfRecBuilder
 
+  wfRecFixpoint : (P : Pred A ℓ) (f : WfRec _<_ P ⊆′ P) {x : A} (q : Acc _<_ x) →
+                  wfRec P f x q ≡ f x (λ y y<x → wfRec P f y (acc-inverse q y y<x))
+  wfRecFixpoint P f (acc rs) = refl
+
```

Comment:
A similar lemma is called `unfold-NAME` in [`Data.List.Properties`](https://agda.github.io/agda-stdlib/Data.List.Properties.html#31652).